src/Makefile

```diff
@@ -112,6 +112,9 @@ CSRC := $(STARTUPSRC) \
         $(SWIFTNAV_ROOT)/src/pps.o \
         $(SWIFTNAV_ROOT)/src/decode.o \
         $(SWIFTNAV_ROOT)/src/signal.o \
+        $(SWIFTNAV_ROOT)/src/ndb.o \
+        $(SWIFTNAV_ROOT)/src/ndb_p1.o \
```

Comment:
I moved src/ndb_p3.o to v2/Makefile.inc. src/ndb_p1.o is common for both boards - is it fine to leave it here?